Makefile

```diff
@@ -1,4 +1,5 @@
 SERVER_URL = http://localhost:8000
```

Comment:
Just a remark: for all the other projects, we kept the loadtest makefile target separated from the main makefile (e.g. a different Makefile file inside the loadtests directory).

I think I like it best to have everything in here, but we should try to be consistent in the future.